### PR TITLE
Hope this works

### PR DESCRIPTION
**Pull request by sometgirldotonline**
Been trying for ages to get my CNAME, MX, and, TXT records to go through.

Sorry if the crazy amount was annoying. Ive given up using the manage webpage.

## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/foxsden.json [hunk 1]
```diff
@@ -4,6 +4,8 @@
         "email": "techseal14@gmail.com"
     },
     "record": {
-        "MX": ["mx1.forwardemail.net"]
+        "MX": "mx1.forwardemail.net",
```
Comment (WilliamDavidHarrison) [inline domains/foxsden.json:7]:
```suggestion
        "MX": ["mx1.forwardemail.net"],
```

File: domains/foxsden.json [hunk 2]
```diff
@@ -4,6 +4,8 @@
...
+        "CNAME": "foxsdenyt.github.io",
```
Comment (WilliamDavidHarrison) [inline domains/foxsden.json:8]: You will need to use A records instead of a CNAME.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/foxsden.json (hunk 2 above)
Comment (FoxsDenYT) [reply domains/foxsden.json:8]: huh but arent a records for ips?

Comment (WilliamDavidHarrison) [reply domains/foxsden.json:8]: Yes, use the GitHub Pages A records.
```
185.199.108.153
185.199.109.153
185.199.110.153
185.199.111.153
```

**Review by sometgirldotonline — COMMENTED** [review]
Updated to use A records and merged WilliamDavidHarrison's changes

**Review by wdhdev — APPROVED** [review]

**Pull request merged**